config/locales/responders.en.yml

```diff
@@ -3,10 +3,10 @@ en:
     actions:
       create:
         notice: '%{resource_name} was successfully created.'
-        # alert: '%{resource_name} could not be created.'
+        alert: '%{resource_name} could not be created.'
       update:
         notice: '%{resource_name} was successfully updated.'
-        # alert: '%{resource_name} could not be updated.'
+        alert: '%{resource_name} could not be updated.'
       destroy:
         notice: '%{resource_name} was successfully destroyed.'
         alert: '%{resource_name} could not be destroyed.'
```

Comment:
Double quotes?

Reply:
Sorry these were still around - not sure why Rubocop ignored these?
Fixed in sha: 7b6cf80